src/main/scala/fi/oph/koski/koskiuser/MockUsers.scala

```diff
@@ -259,6 +259,7 @@ object MockUsers {
       PalveluJaOikeus("KOSKI", Rooli.GLOBAALI_LUKU_TOINEN_ASTE),
       PalveluJaOikeus("KOSKI", Rooli.GLOBAALI_LUKU_KORKEAKOULU),
       PalveluJaOikeus("KOSKI", Rooli.GLOBAALI_LUKU_MUU_KUIN_SAANNELTY),
+      PalveluJaOikeus("KOSKI", Rooli.GLOBAALI_LUKU_TAITEENPERUSOPETUS),
```

Comment:
Pohdin, kun tätä on näihin lisätty, että onko nyt lisätty sen mukaisesti, miten ajatus on, että toimii myös tuotannossa? Ja jos ei ole, niin olisiko selkeintä, että näissä lokaaleissa domain-termeillä nimetyissä mock-käyttäjissä olisi tuotantoa vastaavat oikeudet?

Eli tuleeko oikeudet TPO:hon tuotannossakin tässä yhteydessä eviralle, kelalle ja tilastokeskukselle? Ainakin evira vähän epäilyttää: mihin elintarvikevirasto tarvitsee tietoa tpo-opinnoista?

Reply:
Hyvä pointti! Ei ole keskusteltu miten oikeasti tuotannossa oikeudet annetaan viranomaisille. Käytännössähän roolit annetaan käyttöoikeusryhmille, mutta kaikkia tuotannon käyttöoikeusryhmiä ei suoraan ole mallinnettu Koskeen. Toisaalta koska käyttöoikeusryhmät voivat muuttua, ehkä on ihan hyvä että Koskessa testataan myös muilla roolien yhdistelmillä kuin mitä oikeasti on olemassa.

Tämän muutoksen tein kuitenkin käytännössä siksi, että oli olemassa testi että Eviran käyttäjä voi hakea ja katsella kaikkia opiskeluoikeuksia. Oikeasti tuotannossa ei näytä olevan mitään käyttäjiä juuri Eviralle, ja koko organisaatio on lakkautettu Organisaatiopalvelussa.

Ehdotan että nämä roolit voi kuitenkin jättää näin, tavallaan simuloimaan käyttöoikeusryhmää jolle on annettu kaikkien opiskeluoikeuden tyyppien globaalit lukuoikeudet. Voisin kuitenkin kysyä asiantuntijoilta, miten viranomaisten käyttöoikeudet TPO:n osalta pitäisi toteutua.

Reply:
joo, kuulostaa hyvältä. Ehkä jonkun kommentin voisi laittaa, tai muuttaa muuttujan nimen joksikin muuksi kuin eviraksi?